### ISSUE TITLE
Rework how Qiskit gets cited

### ISSUE THREAD

**Issue by nonhermitian** [Qiskit/qiskit-metapackage#1273]
### What is the expected behavior?
The current bibtex citation format is not ideal, including many user names rather than actual names, and does not take into account the level of contribution to the project.  As such it was decided to replace the current bibtex entry with a contributors file with equivalent information.  For proper citations it was decided to have a general citation, e.g.: "Qiskit 0.26, qiskit.org` , and a proper paper citation to the forthcoming Qiskit paper.

**Comment by boschmitt** [comment]
What is the status on this?

Just today, I had to cite `qiskit` and saw that the bibtex citation format still not ideal:
https://github.com/Qiskit/qiskit/blob/948408b8bdc6937f49bac462e9975254890a6f31/Qiskit.bib#L1-L6

I decided to follow the zenodo to see how the citation would look like when took directly from the DOI. This is what I got:
```latex
@misc{treinish2022,
  title = {Qiskit/Qiskit: {{Qiskit}} 0.34.2},
  shorttitle = {Qiskit/Qiskit},
  author = {Treinish, Matthew and Gambetta, Jay and Nation, Paul and Kassebaum, Paul and {Qiskit-Bot} and Rodr{\'i}guez, Diego M. and De La Puente Gonz{\'a}lez, Salvador and Shaohan Hu and Krsulich, Kevin and Zdanski, Laura and Yu, Jessie and Garrison, Jim and Gacon, Julien and McKay, David and Gomez, Juan and Capelluto, Lauren and {Travis-S-IBM} and Marques, Manoel and Panigrahi, Ashish and Lishman, Jake and Lerongil and Rafey Iqbal Rahman and Wood, Steve and Bello, Luciano and Divyanshu Singh and Drew and Arbel, Eli and Schwarm, Joachim and Daniel, Jonathan and MELVIN GEORGE},
  year = {2022},
  month = feb,
  doi = {10.5281/ZENODO.2573505},
  abstract = {Changelog Changed Increased the {$<$}code{$>$}qiskit-terra{$<$}/code{$>$} version to the latest release 0.19.2 Increased the {$<$}code{$>$}qiskit-aer{$<$}/code{$>$} version to the latest release 0.10.3},
  copyright = {Open Access},
  howpublished = {Zenodo}
}
```

Sincerely, I think this is even worst. `Qiskit` accepts contributions from many developers, thus I would expect something as `Qiskit contributors` as author; for reference, [Cirq](https://zenodo.org/record/5182845#.YhoaNC8w1B0) does exactly that. Instead, I got a list of selected developers (the ones with most contributions?) that even has `Qiskit-Bot` on it.

**Comment by jakelishman** [comment]
I think this issue is covered by the direction the discussion on #934 took, so I'll close it as part of a tidy-up now.  #934 is still waiting to be fixed, but now at least we have #1693 that should do that.
